.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
-.env.*
\ No newline at end of file
+.env.*
+package.json
+package-lock.json
```

Comment:
Both the package.json and package-lock can't be gitignored.
Remove lines 3,4 then you're good to merge!